PSSwagger/PSSwagger.Resources.psd1

```diff
@@ -96,5 +96,8 @@ ConvertFrom-StringData @'
     CouldntFindClientSideParameterSet=Couldn't find client-side parameter set: {0}
     MissingRequiredFilterParameter=Required server-side parameter '{0}' is not required by the client-side, which will cause issues in client-side filtering. Can't include client-side filtering.
     FailedToAddAutomaticFilter=Failed to add automatic client-side filter for candidate command '{0}': Mandatory List parameter '{1}' has no matching Get mandatory parameter. It will be impossible to guarantee execution of the List method before client-side filtering occurs.
+    CopyingUtilityModule=Copying utility module to generated module...
```

Comment:
Why three periods?

Reply:
Cause three periods == an ellipsis, which I use to indicate things are happening (rather than a note or something has happened). I do the same thing when a menu item opens another menu item or dialog rather than actually doing anything (e.g. "File..." -> { "Open...", "Save" }, "Paste", "Cut")

Will remove the three periods.

Reply:
I was not sure what it meant, other actions don't seem to have it (i.e SkippingExistingParameter)